tools/lib/html_branches.py

```diff
@@ -91,8 +91,8 @@ def text(self):
         return s
 
 
-def get_tag_info(token):
-    # type: (Token) -> TagInfo
+def get_tag_info(token,fn=None,master_Idrecord={}):
```

Comment:
I'd like to keep `get_tag_info()` as it is, and perhaps the caller can just use its current output, which already returns a list of ids, to build up its dictionary.